fenix/app/src/main/java/org/mozilla/fenix/experiments/view/ResearchSurfaceSurvey.kt

```diff
@@ -61,50 +99,52 @@ fun ResearchSurfaceSurvey(
     onDismiss: () -> Unit,
     onAccept: () -> Unit,
```

Comment:
Nit: The optional message `title` is not represented here.

Reply:
Looking at the [experiment](https://experimenter.services.mozilla.com/nimbus/viewpoint-mobile-messaging-experiment/summary) I don't see title is being used in the experiment.

Reply:
I see.  currently we've build this to display something that doesn't have a title to it.  I'll create an issue (not blocking the feature) that tracks the work of supporting title in case it's set.  Thanks

Reply:
As agreed.  I've created an [issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1826637) to track this as well.  Thanks